doc/sphinx/source/dbk.rst

```diff
@@ -89,3 +89,18 @@ configuration files of ONNX Runtime:
     cd build
     cmake .. -Donnxruntime_DIR=${HOME}/onnxruntime-bin/lib/cmake/onnxruntime
     make -j$(nproc)
+
+img_color_convert_exp
+_____________________
+
+An experimental DBK that is similar in concept to OpenVX's color convert node.
+It is currently limited to nv12 to RGB, but lays down the groundwork for more
+image manipulation DBKs.
+
+exp_dnn_nms
+___________
+
+An experimental DBK that performs Non-Maximum Suppression (NMS) on a set of
+bounding boxes. It currently makes use of OpenCV's dnn utils. The bounding boxes
+are represented as four cl_int variables in the following order:
+(x, y, width, height). The (0,0) coordinates is the top left.
```

Comment:
> ... (x, y, width, height).

I think we should use the (y1, x1, y2, x2) box presentation which is probably used the most (OpenVINO, ONNX, TensorFlow, Pytorch, TensorRT, TVM). Some of the mentioned frameworks takes boxes in (x1, y1, x2, y2) representation but I think it doesn't affect the results if one happens to pass (y1, x1, y2, x2) boxes to operator that takes (x1, y1, x2, y2) boxes.

> The (0,0) coordinates is the top left.

I think you get same results regardless where the origin is positioned.

```suggestion
(x1, y1, x2, y2).
```

Reply:
The format I used here is how OpenCV declares it's rectangles, but if we are going to use onnx instead for nms, it would make sense to change it and use the representation everyone else uses.